.github/workflows/publish.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
             quay.io/metallb/${{ matrix.image }}
           # generate Docker tags based on the following events/attributes
           tags: |
-            type=ref,event=push
+            type=ref,event=branch
```

Comment:
Will this work when we do a tag?

Reply:
Yes, the other tag type (type=Denver) will generate the server tag

Reply:
sorry, which other tag type?

Reply:
The next line, line 79, `type=semver,pattern={{version}}`

Reply:
An example ci job front docker metadata: https://github.com/docker/metadata-action#semver